src/test/java/org/primefaces/PrimeNGSetupPage.yml

```diff
@@ -0,0 +1,11 @@
+urls:
+  default: "https://www.primefaces.org/primeng/setup"
+
```

Comment:
This isn't necessary if we can't go directly to this page. If we can then we should be going directly here.